.eslintignore

```diff
@@ -0,0 +1,8 @@
+.eslint*
+**/node_modules
+**/bower_components
+**/vendor
+**/*.min.js
+client/data/*
+compiled/
+main.js
```

Comment:
nit: can you add a blank line in the end of files to avoid the "no new line at end of file" github alert?